coins/monero/src/wallet/mod.rs

```diff
@@ -106,6 +108,14 @@ impl ViewPair {
     ViewPair { spend, view }
   }
 
+  pub fn spend(&self) -> EdwardsPoint {
+    self.spend
+  }
+
+  pub fn view(&self) -> Zeroizing<Scalar> {
```

Comment:
I'm unsure we should expose the private key again.  This should return its public key via `&EDWARDS_BASEPOINT_TABLE * self.view.deref()`

Reply:
The whole reason triggered putting these 2 getter functions was that private view key needed to be accesses in some places(when interacting with wallet-rpc) although not at the moment.  I would think those functions are there to give us access to view pair private parts  when needed. Caller can always derive the public key itself when/if needed but other way around isn't possible with only exposing the public key.